Policies/github-privacy-statement.md

```diff
@@ -111,7 +112,7 @@ Co-branding/marketing partners. We may receive information from partners with wh
  
 Publicly available sources. We may also obtain information from publicly available sources as GitHub repositories. 
  
-When you are asked to provide Personal Data, you may decline. And you may use web browser or operating system controls to prevent certain types of automatic data collection. But if you choose not to provide or allow information that is necessary for certain services or features, those services or features may not be available or fully functional.
+When you are asked to provide personal data, you may decline. And you may use web browser or operating system controls to prevent certain types of automatic data collection. But if you choose not to provide or allow information that is necessary for certain services or features, those services or features may not be available or fully functional.
```

Comment:
```suggestion
When you are asked to provide personal data, you may decline. And you may use web browser or operating system controls to prevent certain types of automatic data collection. But if you choose not to provide or allow information that is necessary for certain services or features, those services or features may not be available or fully functional to you.

Reply:
I believe that “for” might be superior to “to” in this instance.

Reply:
👍